.github/workflows/kubeflow-pipelines-integration-v2.yml

```diff
@@ -1,39 +0,0 @@
-name: Kubeflow Pipelines V2 integration Tests
-
-on:
-  push:
-    branches:
-      - master
-  pull_request:
-    paths:
-      - '.github/workflows/kubeflow-pipelines-integration-v2.yml'
-      - '.github/resources/**'
-      - 'samples/core/parameterized_tfx_oss/**'
-      - 'samples/core/dataflow/**'
-      - 'backend/**'
```

Comment:
We need to add these paths to https://github.com/kubeflow/pipelines/blob/095a794cd46df470a821d60b1b5289f2564d33f3/.github/workflows/kfp-samples.yml#L8 so `kfp-samples.yml` is triggered by changes on those.

Reply:
Good call. Done. Thanks, @hbelmiro! 🙏